--- libredwg-0.12/bindings/python/Makefile.am.orig	2021-01-06 19:18:21.747727679 +0100
+++ libredwg-0.12/bindings/python/Makefile.am	2021-01-06 19:20:34.227009977 +0100
@@ -46,7 +46,7 @@
 ## more magic: SWIG_PYTHON_CPPFLAGS resolves to null and python includes
 ## are not passed to gcc via -I
 _LibreDWG_la_CPPFLAGS = -I$(top_srcdir)/include $(AX_SWIG_PYTHON_CPPFLAGS)
-_LibreDWG_la_LDFLAGS = -module -no-undefined $(PYTHON_EXTRA_LDFLAGS)
+_LibreDWG_la_LDFLAGS = -module -avoid-version -shared -no-undefined $(PYTHON_EXTRA_LDFLAGS)
 _LibreDWG_la_LIBADD = ../../src/libredwg.la $(PYTHON_LIBS) $(PYTHON_EXTRA_LIBS)
 
 ## not elegant, but works
